cves/CVE-2015-8213.yml

```diff
@@ -173,17 +173,17 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: Public post describing cve and fix on django development site.
+  name: utils
 interesting_commits:
-  question: |
+  question: | 
     Are there any interesting commits between your VCC(s) and fix(es)?
 
     Write a brief (under 100 words) description of why you think this commit was
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit:
+  - commit: 
```

Comment:
If you did not find any interesting commits, make sure to give a brief summary of other changes to the VVCs to show you did go through their commit history.